Algorithms/Stack/EvaluatePostfixExpression.txt

```diff
@@ -0,0 +1,77 @@
+// Evaluate Postfix Expression
+// Language Used: JAVA
+
+// Given a postfix expression, the task is to evaluate the expression and print the final value. Operators will only include the basic // arithmetic operators like *,/,+ and - . 
+
+// Input format: The first line of input will contains an integer T denoting the no of test cases . Then T test cases follow. Each //	         test case contains an postfix expression.
+
+// Output format: For each test case, in a new line, evaluate the postfix expression and print the value.
+
+// Sample Input :
+// 2
+// 231*+9-
+// 123+*8-
```

Comment:
Use a single large comment for multiple lines than commenting every line

Reply:
Comment section part is done.